gps/cmd.go

```diff
@@ -62,11 +62,6 @@ func (c *monitoredCmd) run(ctx context.Context) error {
 				return &timeoutError{c.timeout}
 			}
 		case <-ctx.Done():
-			if c.cmd.Process != nil {
```

Comment:
Can we wrap this up in a mutex, rather than removing it? Ensuring the underlying process terminates (well...that we send the signal for it, anyway) is an important part of the guarantee, here.